code/game/jobs/job/civilian.dm

```diff
@@ -27,7 +27,7 @@
 	H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/bartender(H), slot_w_uniform)
 	H.equip_to_slot_or_del(new /obj/item/device/pda/bar(H), slot_belt)
 	if(has_alt_title(H, alt_title,"Bartender"))
-		var/obj/item/weapon/permit/gun/bar/permit = new(H)
+		var/obj/item/clothing/accessory/permit/gun/bar/permit = new(H)

```

Comment:
Permits are now accessories? Make sure our permits are up to date with this change.

Reply:
I changed all our permits to follow as they were shooting out compile errors due to not having an owner var.

Reply:
And in the custom_items.txt as well?

Reply:
fuck
I need to change them in custom_items.txt. I doubt that dream daemon got them when I replaced all